app/assets/javascripts/govuk_publishing_components/components/intervention.js

```diff
@@ -0,0 +1,28 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function Intervention ($module) {
+    this.$module = $module
+    this.$banner = this.$module.querySelector('.gem-c-intervention')
+    this.$closeLink = this.$module.querySelector('.gem-c-intervention__dismiss-link')
+  }
+
+  Intervention.prototype.init = function () {
+    this.$module.close = this.handleClose.bind(this)
+
+    if (this.$closeLink) {
+      this.$closeLink.addEventListener('click', this.$module.close)
+    }
+  }
+
+  Intervention.prototype.handleClose = function (event) {
+    if (event) {
+      event.preventDefault()
+    }
+
+    this.$module.style.display = 'none'
```

Comment:
Might be worth checking from an accessibility perspective what's best here - whether to hide the element or remove it from the page.

Reply:
What's the difference with doing this and the way cookie banners does `.hidden = true`? https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js#L105

Reply:
I think the hidden attribute works even if stylesheets are disabled/not loaded, but I'm not certain.

Reply:
Seeing as cookies is the main way we do it on all GOV.UK pages I'm going to follow that convention...

Reply:
Was looking at the wrong place. Here https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js#L67

display = 'none' as well